### ISSUE TITLE
Banner image should have configurable alt attribute

### ISSUE THREAD

**Issue by maxkadel** [pulibrary/static-tables#137]
### Steps to recreate
1. Add a new Banner image for another page, e.g. Undergrad Alumni Index, using the configuration.
2. Note that the alt text for the image, no matter what it is, is "geometric pattern"

### Issues
Images should have meaningful alt text that reflect the image

### User impact
Misleads a user using assistive technology about the display of the page, loses any meaning conveyed by the image, such as a sense of place.

### Recommended fix
Add alt text for banner image to the configuration

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...

**Comment by kevinreiss** [comment]
I believe we've closed this issue in this commit: https://github.com/pulibrary/static-tables/commit/dd4e5fc787aa4670be991ad0bdf087a5cf2602a0.

**Comment by maxkadel** [comment]
In the linked commit the alt text for the banner image is still `alt="geometric pattern"`

**Comment by kevinreiss** [comment]
Does having at least some text (even if it's boilerplate) meet the accessibility requirement?

**Comment by maxkadel** [comment]
I don't think so? I think if the image adds meaning to the page it needs meaningful alt text, and if it does not add meaning to the page it should be hidden from screen readers.

**Comment by christinach** [comment]
The allt attribute should describe the image. If the image is not a geometric pattern it shouldn't be geometric pattern. I agree that we should try to configure it in a config file for all the sites.
